docs/data/charts/components/BasicBarDemo.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { ResponsiveChartContainer, LinePlot, useDrawingArea } from '@mui/x-charts';
+
+function DrawingAreaBox() {
+  const { left, top, width, height } = useDrawingArea();
+  return (
+    <React.Fragment>
+      <path
+        d={`M ${left} ${top} l ${width} 0 l 0 ${height} l -${width} 0 Z`}
+        style={{ fill: 'none', stroke: 'black' }}
```

Comment:
All the demos don't look the best (basically illegible) in dark mode. 🙈 
Is there an option to access the `theme` here and use the `contrastText` or some other dynamic color so that these charts behave more like the original charts that we expose?